sklearn/utils/estimator_checks.py

```diff
@@ -1318,7 +1326,7 @@ def check_class_weight_balanced_linear_classifier(name, Classifier):
 
 
 def check_estimators_overwrite_params(name, Estimator):
-    X, y = make_blobs(random_state=0, n_samples=9)
+    X, y = make_blobs(random_state=0, n_samples=9, n_features=3)
```

Comment:
For bicluster classes, there is singular vector ``vt`` of shape ``[n_features, n_components]`` on which [k_means](https://github.com/maniteja123/scikit-learn/blob/bf459f445c43a663df565bc0aa237f57acfc7264/sklearn/cluster/bicluster.py#L480) is applied for which the default value for ``n_clusters`` is 3 which should be greater than equal to the number of samples of ``vt``, which is ``n_features``(which has default of 2 in ``make_blobs``), hence changed it to 3. And the other estimators were still passing, so I hoped it would be okay.